Assets/Scripts/PlayerController.cs

```diff
@@ -1,12 +1,18 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.InputSystem;
 
 public class PlayerController : MonoBehaviour
 {
     [SerializeField] float moveSpeed = 10f;
     [SerializeField] float jumpForce = 10f;
+
+    [Header("Jumping")]
+    [SerializeField] private TriggerCollision jumpDetector;
```

Comment:
When using the `[SerializeField]` attribute is not necessary to use the `private` access modifier because the data types without the `public` modifier are private by default

Reply:
Useful. It looks better without the 'private' modifier, I will try it.